src/lib/WIFI/devWIFI.cpp

```diff
@@ -718,6 +726,23 @@ static void WebUploadForceUpdateHandler(AsyncWebServerRequest *request) {
   }
 }
 
+#ifdef HAS_WIFI_JOYSTICK
+static void WebUdpControl(AsyncWebServerRequest *request)
+{
+  const String &action = request->arg("action");
+  if (action.equals("joystick_begin"))
+  {
+    WifiJoystick::StartSending(request->client()->remoteIP(),
+      request->arg("interval").toInt(), request->arg("channels").toInt());
+    request->send(200, "text/plain", "ok");
+  }
+  else if (action.equals("joystick_end"))
+  {
+    WifiJoystick::StopSending();
```

Comment:
Oops, this is my bad. `joystick_end` is missing any response data so when end is called it just sends back an empty response. Can you add a `request->send(200, "text/plain", "ok");`?